src/Controls/samples/Controls.Sample/Startup.cs

```diff
@@ -64,7 +64,7 @@ public void Configure(IAppHostBuilder appBuilder)
 				})
 				.ConfigureFonts(fonts =>
 				{
-					fonts.AddFont("Dokdo-Regular.ttf", "Dokdo");
+					fonts.AddFont("Dokdo.ttf", "Dokdo");
```

Comment:
Seems to be an issue in the registrar not looking at files. I'll try get it fixed asap.